docs/internals/contributing/writing-code/javascript.txt

```diff
@@ -0,0 +1,75 @@
+==========
+JavaScript
+==========
+
+Django includes JavaScript tests which may be run independently of the Python
```

Comment:
_may be run independently_ -- this suggests that JS test can also be run as part of Python test. How about:

_Django includes JavaScript tests that can easily be run in a browser or, after installing dependencies, in command line._